Error building project issue with docker container cache access

Ran into the following error while building the project for testing. The following command and output reliably reproduces for me.

```
$make clean all
rm -rf bin cover.html cover.out bin/pullcord bin/genhash
rm -f `for file in ; do \
	find . -name ${file}; \
done`
docker run -it --rm -v ${PWD}/.build_gopath/:/go/ -v ${PWD}/:/go/src/github.com/stuphlabs/pullcord/ -w /go/src/github.com/stuphlabs/pullcord/ -u `id -u` golang:latest go test -v -coverprofile cover.out -covermode set ./...
failed to initialize build cache at /.cache/go-build: mkdir /.cache: permission denied
Makefile:57: recipe for target 'cover.out' failed
make: *** [cover.out] Error 1
```

--- a/Makefile.in
+++ b/Makefile.in
@@ -15,6 +15,7 @@ GO := ${DOCKER} \
 	-v $${PWD}/:/go/src/${ROOTPKG}/ \
 	-w /go/src/${ROOTPKG}/ \
 	-u `id -u` \
+	-e XDG_CACHE_HOME='/tmp/.cache' \
 	_DOCKER_GOLANG_IMG_ \
 	_GO_CMD_
 ],[

